SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityUsagePoller.java

```diff
@@ -280,6 +280,14 @@ public void runActionOnPoll() {
     }
   }
 
+  private boolean isEligibleForShuffle(SingularityTaskId task) {
+    return (
+        isLongRunning(task)
+            && !configuration.getDoNotShuffleRequests().contains(task.getRequestId())
+            && (TimeUnit.MILLISECONDS.toMinutes(System.currentTimeMillis() - task.getStartedAt()) > configuration.getMinutesBeforeNewTaskEligibleForShuffle())
```

Comment:
I wonder if we should go by the task running timestamp instead? Take out any delay that may have been caused by mesos/singularity that way